### DISCUSSION TITLE
Issue visualising data in newer than v5.7 paraview

### DISCUSSION THREAD

**Discussion by Elphikaus** [Chaste/Chaste#188] [Q&A] [answered]
I use cell-based chaste. Specifically a project that uses capsules. I can not visualise in paraview with my pvsm file state as I get an error that says

```
Warning: In ./Remoting/ServerManager/vtkSMProxy.cxx, line 895
vtkSMSourceProxy (0x561902f68ad0): Proxy (filters, GlyphLegacy)  has been deprecated in ParaView 5.6 and will be removed by ParaView 5.7. 
        It has been replaced by 'GlyphWithCustomSource'. Please consider using that instead.
```

Is this error other people get?

**Comment by BJackal** [comment]
Could you please provide either a paraview data file or the code that produced it. I do not have issues with paraview version 5.10 so I suspect it could be an issue with data.

**Comment by Elphikaus** [comment]
I uploaded the pvsm file along with a sample script and some data that relates to the pvsm file.

The script that produced the data is part of a project I have been working on. But I did not include that, if you need more to offer the best feedback, let me know.
[chaste-paraview.zip](https://github.com/Chaste/Chaste/files/13350516/chaste-paraview.zip)

**Comment by BJackal** [comment, marked as answer]
I get a similar warning error about the depreciated glyph type but can visualise the simulation itself. Without knowing how this pvsm file is generated though Im unsure at what point the glyphs are generated. The end state looks like this.

![Paraview_Image_1](https://github.com/Chaste/Chaste/assets/44051158/f230a3bc-0d86-40a0-8cca-4d7bd50a9992)

Are you unable to visualise the simulation at all ?

**Reply by Elphikaus** [reply]
I can visualise now. I needed to update the calculator.
